test/plots/index.js

```diff
@@ -284,6 +284,7 @@ export * from "./heatmap.js";
 export * from "./legend-color.js";
 export * from "./legend-opacity.js";
 export * from "./legend-symbol.js";
+export * from "./raster-blue-noise.js";
```

Comment:
It's not actually blue noise though, so perhaps better avoid calling it that even just in the file name. Otherwise someone will end up thinking it is.